completion/yathesis.cwl

```diff
@@ -182,8 +250,8 @@ juniorresearcher*
 #endkeyvals
 #
 # Misc
-\ordernumber[numéro d'ordre]#n
-\ordernumber#*n
+\ordernumber[%<numéro d'ordre%>]#n
+\ordernumber#n*
```

Comment:
I think, it needs to be #*n , in that order

Reply:
> I think, it needs to be #*n , in that order

OK, done.